yb-voyager/src/srcdb/postgres.go

```diff
@@ -4,15 +4,23 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
-	"strings"
 	"net/url"
+	"os/exec"
+	"regexp"
+	"strconv"
+	"strings"
 
 	"github.com/jackc/pgx/v4"
 	log "github.com/sirupsen/logrus"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/datafile"
 	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
 )
 
+var PgDumpPath string
+var PgRestorePath string
+
+const PG_COMMAND_VERSION float64 = 14.0
```

Comment:
Please, use strings for version comparison. What if there is a `14.2.1` version?

Reply:
Done